.README/INSTALL.md

```diff
@@ -46,8 +47,47 @@ You can then activate `2.5D Isometric Map` addon int `Extensions` tab:
 Isometric map addon is now installed on your project. You can now visit [user guide](./USER_GUIDE.md) to start design
 your world !
 
+## GDNS Inheritance Workaround
+
+If you encounter the issue [above](https://github.com/godotengine/godot/issues/38866), the following manual steps should done
+to allow the addon to function correctly. Please note this is temporary and should hopefully be resolved permanently at some
+point in the future so users won't have to do this manual process.
+
+After installing the addon using the steps above, find the `res://addons/IsometricMap/GdnsScripts/` directory and open each
+`.gdns` file located within. This should bring up the file properties in the inspector. For each file, proceed to check,
+and then uncheck, the `Singleton` property.
+![gdns workaround]
+
+Make sure to do this for the following files for safety:
```

Comment:
Not sure it is necessary to list them, as the user need to do it on all gdns file. As an example some GDNS will be added with collision work. So it would be necessary to add each gdns in readme too.

Reply:
Okay, will remove!